app/.editorconfig

```diff
@@ -20,25 +20,15 @@ indent_size = 4
 # HTML-Files
 [*.html]
 indent_style = tab
-indent_size = 4
-
-# TypoScript
-[{*.ts,*.setupts,*.constantsts,*.tsc,*.ts1}]
-indent_style = tab
-indent_size = 4
+indent_size = 2
```

Comment:
Did you just overwrite .editorconfig with the one from core? When yes, mine one is based on the core's with additionals. and I personally don't like only 2 indents for typoscript and html. Is it ok for you?